glotaran/analysis/scheme.py

```diff
@@ -37,64 +37,70 @@ def __init__(self,
         self.nfev = nfev
 
     @classmethod
-    def from_yml_file(cls, filename: str) -> 'Scheme':
+    def from_yml_file(cls, filename: str) -> "Scheme":
 
         try:
             with open(filename) as f:
                 try:
                     scheme = yaml.load(f, Loader=yaml.FullLoader)
                 except Exception as e:
-                    raise Exception(f"Error parsing scheme: {e}")
+                    raise ValueError(f"Error parsing scheme: {e}")
         except Exception as e:
-            raise Exception(f"Error opening scheme: {e}")
+            raise OSError(f"Error opening scheme: {e}")
```

Comment:
@s-weigand Are all this changes in the exceptions done by you or py black?

Reply:
The Exception changes are all done by me (only in this case pyupgrade changed my `IOError` to an `OSError`), so sonar cloud complains about less code smells see https://github.com/s-weigand/pyglotaran/pull/14 and #409 .

Reply:
For the `OSError` see the changes in python 3.3 https://docs.python.org/3/library/exceptions.html#OSError.filename2